frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -440,28 +433,25 @@ const QueueItem = ({
         variables: {
           id: internalId,
           deviceId: props.deviceId,
-          result: props.result,
+          results: props.results,
           dateTested: props.dateTested,
           deviceSpecimenType: props.deviceSpecimenType,
         },
       })
         .then((response) => {
           if (!response.data) throw Error("updateQueueItem null response");
           updateDeviceSpecimenTypeId(
-            response.data.editQueueItem.deviceSpecimenType.internalId
+            response?.data?.editQueueItemMultiplex?.deviceSpecimenType
+              ?.internalId ?? ""
           );
-          updateTestResultValue(
-            response.data.editQueueItem.result || undefined
-          );
-
           updateTimer(
             internalId,
-            response.data.editQueueItem.deviceSpecimenType.deviceType
-              .testLength as number
+            response?.data?.editQueueItemMultiplex?.deviceSpecimenType
+              ?.deviceType.testLength as number
           );
           updateDeviceTestLength(
-            response.data.editQueueItem.deviceSpecimenType.deviceType
-              .testLength as number
+            (response?.data?.editQueueItemMultiplex?.deviceSpecimenType
+              ?.deviceType?.testLength as number) ?? 0
```

Comment:
Is this the default if no `testLength` is found? If so I would update to 15 instead of 0, so the timer continues to work.

Reply:
yeah, the linter was giving me errors because of a possible null/undefined scenario. I will update this to 15. Thanks for catching this!